docs/test_scenarios.rst

```diff
@@ -0,0 +1,729 @@
+              created entry and passing an invalid value for the ``--vault``
+              option.
+:steps: Run ``rho auth clear --name <name> --vault <invalidvault>``
+:expectedresults: The auth entry is not removed and the ``credentials`` file is
+                  not updated.
+:bug: https://github.com/quipucords/rho/issues/132
+
+Clear multiple auth entries with vault (negative)
+-------------------------------------------------
+
+:description: Clear multiple auth entries using the ``--all`` option and
+              passing an invalid value for the ``--vault`` option.
+:steps: Run ``rho auth clear --all --vault <invalidvault>``
+:expectedresults: All auth entries are not removed and the ``credentials`` file
+                  is not deleted.
+
+rho auth edit
+=============
+
+Edit auth username
+------------------
+
+:description: Edit the username of an auth entry.
+:steps: Run ``rho auth edit --name <name> --username <newusername>``
+:expectedresults: The auth username must be updated and the ``credentials``
+                  file must be updated.
+
+Edit auth password
+------------------
+
+:description: Edit the password of an auth entry.
+:steps: Run ``rho auth edit --name <name> --password <newpassword>``
+:expectedresults: The auth password must be updated and the ``credentials``
+                  file must be updated.
+
+Edit auth sshkeyfile
+--------------------
+
+:description: Edit the sshkeyfile of an auth entry.
+:steps: Run ``rho auth edit --name <name> --sshkeyfile <newsshkeyfile>``
+:expectedresults: The auth sshkeyfile must be updated and the ``credentials``
+                  file must be updated.
+
+Edit auth username with vault
+-----------------------------
+
+:description: Edit the username of an auth entry passing the ``--vault``
+              option.
+:steps: Run ``rho auth edit --name <name> --username <newusername> --vault
+        <vault>``
+:expectedresults: The auth username must be updated and the ``credentials``
+                  file must be updated.
+
+Edit auth password with vault
+-----------------------------
+
+:description: Edit the password of an auth entry passing the ``--vault``
+              option.
+:steps: Run ``rho auth edit --name <name> --password <newpassword> --vault
+        <vault>``
+:expectedresults: The auth password must be updated and the ``credentials``
+                  file must be updated.
+
+Edit auth sshkeyfile with vault
+-------------------------------
+
+:description: Edit the sshkeyfile of an auth entry passing the ``--vault``
+              option.
+:steps: Run ``rho auth edit --name <name> --sshkeyfile <newsshkeyfile> --vault
+        <vault>``
+:expectedresults: The auth sshkeyfile must be updated and the ``credentials``
+                  file must be updated.
+
+Edit auth username (negative)
+-----------------------------
+
+:description: Try to edit the username of a not created n auth entry.
```

Comment:
several cases of `not created n auth entry` that I think just need the `n ` removed.